docker-compose.yml

```diff
@@ -246,6 +351,13 @@ services:
           interval: 10s
           timeout: 10s
           retries: 60
+        deploy:
+            resources:
+                limits:
+                    cpus: '0.50'
+                    memory: 50M
```

Comment:
Hm, what does the `platform` container actually run? it probably runs a webserver right, in which case this is probably rather low, especially if it has to build the webpack bundle. cc @juhah @tumppi @mondoreale

Reply:
`platform` runs nginx serving prebuilt app bundle so no webpack is needed and should not need much resources.

`network-explorer` on the other hand uses webpack dev server currently so it needs more resources. We can get rid of webpack and use nginx instead when we have enough runway to implement this.